swarm/tools/runs_gc.py

```
@@ -72,15 +73,23 @@ def age_days(self) -> float:
+    """
     total = 0
     try:
-        for entry in path.rglob("*"):
-            if entry.is_file():
+        for entry in os.scandir(path):
+            if entry.is_file(follow_symlinks=False):
                 try:
-                    total += entry.stat().st_size
+                    total += entry.stat(follow_symlinks=False).st_size
                 except OSError:
                     pass
```

Ruff (0.15.6)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)